cmd/bblfsh/client.go

```diff
@@ -39,10 +40,19 @@ func (c *clientCmd) Execute(args []string) error {
 		run = c.runStandalone
 	}
 
+	var encoding protocol.Encoding
+	switch(c.Encoding) {
+	case "Base64":
+		encoding = protocol.Base64
+	default:
+		encoding = protocol.UTF8
```

Comment:
Sorry, from https://github.com/bblfsh/sdk/pull/131#issuecomment-309488209 I misunderstood that the encoding was optional at the protocol level, but I see in the implementation that it is mandatory, then it doesn't make sense to leave it undefined.